### PR TITLE
Add URI calls for app and system and the PB version of the PIN API.

### PR DESCRIPTION
**Pull request by ababut**

### CODE REVIEW

File: api/blackberry_app.js [hunk 1]
```diff
@@ -175,4 +180,58 @@ blackberry.app ={
...
-};
\ No newline at end of file
+	
+	
+	/**
+	 * @uri 
+	 * @PB10
+	 * @returns {JSON of all the properties}
+	 * {
+	 * 	"data":{
```
Comment (jeffheifetz) [inline api/blackberry_app.js:190]: Shouldn't you have code and msg here?

Comment (jeffheifetz) [inline api/blackberry_app.js:190]: Based on pull #107 I guess you should ignore this

**Comment by jeffheifetz** [comment]
Great, this looks ready to merge

**Comment by ababut** [comment]
Thanks! I am proposing that we should only show the API-specific data in each API's docs. The rc/message parts are redundant, they can be documented in a common spot together with the HTTP return codes.

**Pull request merged**